### PR TITLE
Implemented: logger using vue logger plugin(#103)

### PR DESCRIPTION
**Pull request by Ritika-Patel08**
### Related Issues

#103 

### Short Description and Why It's Useful

Added vue-logger-plugin dependency and use logger instead of console for errors.

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/import#contribution-guideline)

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -12,7 +13,6 @@ import '@ionic/vue/css/core.css';
 import '@ionic/vue/css/normalize.css';
 import '@ionic/vue/css/structure.css';
 import '@ionic/vue/css/typography.css';
-
```
Comment (amansinghbais) [inline src/main.ts:15]: Remove Unwanted changes

File: src/registerServiceWorker.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { register } from 'register-service-worker'
-
```
Comment (amansinghbais) [inline src/registerServiceWorker.ts:2]: Remove Unwanted changes

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]
Update the PR with main branch and check once again for all components.

File: src/registerServiceWorker.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { register } from 'register-service-worker'
```
Comment (ymaheshwari1) [inline src/registerServiceWorker.ts:1]: Remove this file and its import.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/store/modules/util/actions.ts [hunk 4]
```diff
@@ -6,6 +6,8 @@ import * as types from './mutation-types'
 import { showToast } from '@/utils'
 import { hasError } from '@/adapter'
 import { translate } from '@hotwax/dxp-components'
+import { UserService } from '@/services/UserService'
```
Comment (ymaheshwari1) [inline src/store/modules/util/actions.ts:9]: Unused import

File: src/views/CreateUser.vue [hunk 5]
```diff
@@ -218,13 +219,9 @@ export default defineComponent({
         } else {
           throw resp.data;
         }
-      } catch (err:any) {
-        let errorMessage = translate('Failed to create user.');
-        if (err?.response?.data?.error?.message) {
-          errorMessage = err.response.data.error.message
-        }
```
Comment (ymaheshwari1) [inline src/views/CreateUser.vue:225]: We need to keep this changes.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/store/modules/util/actions.ts (hunk 4 above)
Comment (Ritika-Patel08) [reply src/store/modules/util/actions.ts]: Removed this import.

File: src/views/CreateUser.vue (hunk 5 above)
Comment (Ritika-Patel08) [reply src/views/CreateUser.vue]: Reverted. 
You can review.

File: src/store/modules/user/actions.ts [hunk 6]
```diff
@@ -80,8 +81,8 @@ const actions: ActionTree<UserState, RootState> = {
       // If any of the API call in try block has status code other than 2xx it will be handled in common catch block.
       // TODO Check if handling of specific status codes is required.
       showToast(translate('Something went wrong while login. Please contact administrator'));
-      console.error("error", err);
-      return Promise.reject(err instanceof Object ? err :new Error((err)))
```
Comment (ymaheshwari1) [inline src/store/modules/user/actions.ts:84]: We need to keep this change as well.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

**Pull request merged**